app/models/submission/scrna_core_cdna_prep_feasibility_calculator.rb

```diff
@@ -3,25 +3,71 @@
 # This module provides methods for calculating the full allowance volume and the
 # final resuspension volume for scRNA core cDNA prep feasibility validations.
 module Submission::ScrnaCoreCdnaPrepFeasibilityCalculator
-  # This method calculates the full allowance volume (in microlitres) for the
-  # specified number of cells per chip well, which is typically specified in
-  # in a bulk submission per study and project. It uses the pooling settings
-  # from the scRNA config. It first calculates the chip loading volume for the
-  # given number of cells per chip well, and then the full allowance for that
-  # chip loading volume.
+  SCRNA_CORE_CDNA_PREP_GEM_X_5P = 'Limber-Htp - scRNA Core cDNA Prep GEM-X 5p'
+  # Constants for the headers in the bulk submission for scRNA core cDNA prep.
+  HEADER_BARCODE = 'barcode' unless defined?(HEADER_BARCODE)
```

Comment:
Instead of checking for defined you can use `or-equals` e.g. `HEADER_BARCODE ||= 'barcode'`.